tests/test_mixins.py

```diff
@@ -45,3 +45,12 @@ def test_channel_summary_mixin_empty():
     assert mixin.modifiers == []
     assert mixin.parameters == []
     assert mixin.samples == []
+
+
+def test_channel_nbins_sorted_as_channels(spec):
+    assert "channels" in spec
+    spec["channels"].append(spec["channels"][0].copy())
+    spec["channels"][-1]["name"] = "a_make_first_in_sort_channel2"
```

Comment:
@kratsg there is probably a more elegant way to do this, so please give recommendations and ask for changes here.